app/package.json

```diff
@@ -3,14 +3,18 @@
   "dependencies": {
     "@aws-sdk/client-s3": "^3.523.0",
     "@aws-sdk/s3-request-presigner": "^3.523.0",
+    "@chakra-ui/react": "2.8.2",
     "@faker-js/faker": "8.3.1",
     "@google-analytics/data": "4.1.0",
     "@headlessui/react": "1.7.13",
     "@tailwindcss/forms": "^0.5.3",
     "@tailwindcss/typography": "^0.5.7",
+    "@tanstack/react-form": "0.27.0",
     "apexcharts": "^3.41.0",
     "clsx": "^2.1.0",
+    "dompurify": "3.1.6",
     "headlessui": "^0.0.0",
+    "jsdom": "^25.0.0",
```

Comment:
If I understand correctly version of jsdom is critical for safe operation of dompurify on the server, so we would probably need to keep an eye on it in the future, and upgrade it if some new vulnerabilities are found.